src/pages/instances/InstanceOverviewMetrics.tsx

```diff
@@ -30,6 +34,14 @@ const InstanceOverviewMetrics: FC<Props> = ({ instance, onFailure }) => {
 
   const instanceMetrics = getInstanceMetrics(metrics, instance);
 
+  if (isRestricted) {
+    return (
+      <div className="u-text--muted">
+        Usage details unavailable for restricted users
```

Comment:
"Usage" is already the title of the section, so maybe it's redundant here:
```suggestion
        Details unavailable for restricted users
```